trillian/ctfe/handlers_test.go

```diff
@@ -250,19 +253,20 @@ func TestPostHandlersFailure(t *testing.T) {
 	info := setupTest(t, []string{cttestonly.FakeCACertPEM}, nil)
 	defer info.mockCtrl.Finish()
 	for path, handler := range info.postHandlers() {
-		s := httptest.NewServer(handler)
-		defer s.Close()
+		t.Run(path, func(t *testing.T) {
+			s := httptest.NewServer(handler)
 
-		for _, test := range tests {
-			resp, err := http.Post(s.URL+"/ct/v1/"+path, "application/json", test.body)
-			if err != nil {
-				t.Errorf("http.Post(%s,%s)=(_,%q); want (_,nil)", path, test.descr, err)
-				continue
-			}
-			if resp.StatusCode != test.want {
-				t.Errorf("http.Post(%s,%s)=(%d,nil); want (%d,nil)", path, test.descr, resp.StatusCode, test.want)
+			for _, test := range tests {
```

Comment:
Should this be more like:
```golang
for path, handler := range ... {
  for _, test := range tests {
    t.Run(fmt.Sprintf("%s/%s", path, test.descr), func(...
  }
}
```
?

If so, then then the `Errorf / continue` can also become a `Fatalf` :)

Reply:
That one wasn't flagged by inspection analysis but seems like it could be improved. I'll have a look.

Reply:
Done.